csafe_cmd.py

```diff
@@ -122,24 +122,25 @@ def write(arguments):
 
     #check for frame size (96 bytes)
     if len(message) > 96:
-        print "Message is too long: " + len(message)
+        print("Message is too long: " + len(message))
 
     #report IDs
     maxmessage = max(len(message) + 1, maxresponse)
 
     if maxmessage <= 21:
         message.insert(0, 0x01)
         message += [0] * (21 - len(message))
-    elif maxmessage <= 63:
-        message.insert(0, 0x04)
-        message += [0] * (63 - len(message))
+    ## see https://github.com/wemakewaves/PyRow/issues/5
+    #elif maxmessage <= 63:
+    #    message.insert(0, 0x04)
+    #    message += [0] * (63 - len(message))
```

Comment:
This change (i.e. just the commit this was in) can be straight up cherry picked onto the py3row codebase with no merge conflicts

Thanks for keeping the commits organized and separated btw!